cryptofeed/standards.py

```diff
@@ -116,6 +116,7 @@ def timestamp_normalize(exchange, ts):
     },
     TRADES: {
         DYDX: 'v3_trades',
+        HITBTC: 'subscribeTrades',
```

Comment:
this looks like you've duplicated it. I can see below on line 122 that its already defined. Please remove this

Reply:
I missed this one. Now fixed with latest push